src/backend/TrafficCourts/Common/Features/Mail/Model/MailTemplateCollection.cs

```diff
@@ -66,6 +66,31 @@ public class MailTemplateCollection
                 HtmlContentTemplate = "TBD",
                 PlainContentTemplate = "TBD"
             },
+        new MailTemplate()
+            {
+                TemplateName = "DisputantUpdateRequestReceivedTemplate",
+                Sender = "DoNotReply@tickets.gov.bc.ca",
```

Comment:
only one of HtmlTemplate Or PlainContentTemplate should be filled in.

Reply:
the other should be blank.

Reply:
Please clarify this for Colm so he knows when he fills out this data (TCVP-2017).

Reply:
ah ok thanks